TileFormats/FeatureTable/schema/featureTable.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema#",
+    "id" : "featureTable.schema.json",
+    "title" : "Feature Table",
+    "type" : "object",
+    "description" : "A set of semantics containing per-tile and per-feature values defining the position and appearance properties for features in a tile.",
+    "patternProperties" : {
```

Comment:
To elaborate, a semantic value is currently defined as either an object with a `byteOffse`t referencing the binary, an array, or a single value, and I think we currently forbid using SEMANTICS not defined for a tile format, so if it is an object, I think you should always have a `byteOffset`.